app/components/cms_resource_card_component/cms_resource_card_component.scss

```diff
@@ -0,0 +1,41 @@
+.cms-resource-card {
+  &_wrapper {
```

Comment:
It appears cms-resource-card as a class is never used, and _wrapper is just appended on the end. Do we need the wrapper, can we not just have the `cms-resource-card` be the main class?